conda_smithy/templates/azure-pipelines-linux.yml.tmpl

```
@@ -33,6 +33,14 @@ jobs:
          sudo docker image prune --all --force
          df -h
     displayName: Manage disk space
+{%- endif %}
+{%- if not (azure.swapfile_size|string).startswith("0") %}
```

Think this needs an update too

```suggestion
{%- if not (azure.settings_linux.swapfile_size|string).startswith("0") %}
```